File: C/Exercicio20.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  float p1, p2, media;
  
  printf("Digite a nota da P1: ");
  scanf("%f", &p1);
 
  printf("Digite a nota da P2: ");
  scanf("%f", &p2);
 
  media = (p1+(p2*2))/3;
 
  if (media >= 5){
      printf("Aprovado!");
  }
  else{
      printf("Reprovado!");
  }
  
  return 0;
}

File: Python/Exercicio20.py
p1 = float(input("Digite a nota da P1: "))
p2 = float(input("Digite a nota da P2: "))
 
m = (p1+(2*p2))/3
 
if (m >= 5):
    print(f"Sua média foi {m:.2f} e você está aprovado!")
else:
    print("Reprovado!")